frontend/src/app/gaming/gaming.component.scss

```diff
@@ -0,0 +1,12 @@
+app-link {
+    height: 200px;
+    font-size: 1.5rem;
+
+    &[routerLink="/gaming/larp"] {
```

Comment:
It would feel better to me to add a class to that app-link than to look at the routerLink. I know it's unlikely that the link will change, but this is a code smell for me.

Reply:
yup sounds good. will do